src/css/components/Breadcrumbs.scss

```diff
@@ -0,0 +1,33 @@
+@import "../config/index.scss";
+
+// * @define default Breadcrumbs
+
+.Nav-breadcrumb {
+  color: black;
+  font-size: 16px;
+}
+
+a {
+  color: black;
+}
+
+.Breadcrumb {
+  li {
+    display: inline-block;
+    margin-right: 10px;
+    position: relative;
+
+    &:last-child {
+      color: #5d7180;
```

Comment:
I think this is also supposed to be `color-neutral(7)` but am not 100% positive.

Reply:
Done!